src/plugins/irc-events/connection.js

```diff
@@ -0,0 +1,75 @@
+var _ = require("lodash");
+var identd = require("../../identd");
+var Msg = require("../../models/msg");
+
+module.exports = function(irc, network) {
+	var client = this;
+
+	client.emit("msg", {
+		chan: network.channels[0].id,
+		msg: new Msg({
+			text: "Network created, connecting to " + network.host + ":" + network.port + "..."
+		})
+	});
+
+	irc.on("raw socket connected", function() {
+		identd.hook(irc.connection.socket, network.username);
```

Comment:
Looks like this was done to support node 0.12, same thing here:
https://github.com/kiwiirc/irc-framework/blob/3f22b630a0109679b0479a3be97d766a026d7cff/src/connection.js#L107-L109